conmon-rs/server/src/config.rs

```diff
@@ -166,6 +166,26 @@ pub enum Commands {
         #[clap(long("uts"))]
         /// Unshare the UTS namespace.
         uts: bool,
+
+        #[clap(
+            long("uid-mappings"),
+            required_if_eq("user", "true"),
+            short('u'),
+            value_name("MAPPINGS")
+        )]
+        /// User ID mappings for unsahring the user namespace.
+        /// Allows multiple mappings in the format: "CONTAINER_ID HOST_ID SIZE".
+        uid_mappings: Vec<String>,
+
+        #[clap(
+            long("gid-mappings"),
+            required_if_eq("user", "true"),
+            short('g'),
+            value_name("MAPPINGS")
```

Comment:
This value_name is the same as line 174.

Reply:
Yeah I guess we can just remove that and use the default. Fixed as suggested.